lib/xboxrt/vcruntime/purecall.c

```diff
@@ -0,0 +1,37 @@
+#include <assert.h>
+#include <stdbool.h>
+#include <stdlib_ext_.h>
+#include <windows.h>
+#include <hal/debug.h>
+
+_purecall_handler current_purecall_handler = NULL;
+
+int __cdecl _purecall ()
+{
+    if (current_purecall_handler) {
+        current_purecall_handler();
+    }
+
+    DbgPrint("Pure virtual function called!\n");
+    debugPrint("Pure virtual function called!\n");
+
+    // Make sure the handler doesn't return
+    while(true) {
+        Sleep(1000);
+    }
+
+    // return value is not documented my MS
+    return 0;
```

Comment:
Should document that this is currently unreachable, too?

Reply:
Done.